emailmonitor/oxa_email_monitor.py

```diff
@@ -167,16 +167,26 @@ def process_error_line( cursor, line ):
             cursor.execute("UPDATE oxa.oxa_activationsummary SET failed=failed+1 WHERE  xyear="+y+" and xmonth="+mo+" and xday="+d)    
              
         
-
-    
+# Get the IP list of edxapp VMs dynamically
+def get_list_of_vm_ips():
+    ip_list = []
+	os.system('nmap -sP 10.0.0.1-255 | grep -oE "10.0.0.([0-9]{1,3})" > vm_ip_list.txt')
+    with open('vm_ip_list.txt') as f:
```

Comment:
But this is still better than hard coded IP addresses. We can revisit this latter for the concerns.